README.md

```diff
@@ -67,10 +67,10 @@ response in a text view.
 You can build the application by running:
 
 ```
-$ bazel build //ios-app:GetUrl
+$ bazel build ios-app --ios_sdk_version=8.4
```

Comment:
I'm not entirely sure - it is possible I indeed updated my xcode since my earlier revision. iOS SDK 8.4 is supported in XCode 6.4, which at the time of this writing seems to be apple's current stable version. I've updated the readme to somewhat explain this fact, is this sufficient?